setup.py

```diff
@@ -63,19 +64,23 @@ def initialize_options(self):
         # Default to /usr/local for Homebrew
         prefix = '/usr/local'
       else:
-        print('Warning: mfg-inspector output is not fully implemented for '
-              'Windows. OpenHTF will be installed without it.')
-        self.skip_proto = True
+        prefix = None
 
-    maybe_protoc = os.path.join(prefix, 'bin', 'protoc')
-    if os.path.isfile(maybe_protoc) and os.access(maybe_protoc, os.X_OK):
+    self.protoc = None
+    if prefix:
+      maybe_protoc = os.path.join(prefix, 'bin', 'protoc')
+      if os.path.isfile(maybe_protoc) and os.access(maybe_protoc, os.X_OK):
         self.protoc = maybe_protoc
-    else:
+      else:
         print('Warning: protoc not found at %s' % maybe_protoc)
         print('setup will attempt to run protoc with no prefix.')
-        self.protoc = 'protoc'
+    if not self.protoc:
+      self.protoc = find_executable('protoc')
+      pc_path = os.path.dirname(find_executable('protoc'))
+      self.protodir = os.path.abspath(os.path.dirname(pc_path) + '/../lib')
```

Comment:
Prefer using os.path.join instead of hard-coding path separators.

Reply:
Done